frontend/src/pages/Login/Login.jsx

```diff
@@ -7,16 +7,34 @@ const uiConfig = {
   signInOptions: [firebase.auth.GoogleAuthProvider.PROVIDER_ID],
   callbacks: {
     signInSuccessWithAuthResult: (authResult, redirectUrl) => {
-      console.log(authResult); // eslint-disable-line no-console
+      console.log(authResult.credential.idToken); // eslint-disable-line no-console
       console.log(redirectUrl); // eslint-disable-line no-console
-      console.log(firebase.auth().currentUser.getIdToken(false)); // eslint-disable-line no-console
-      return false;
+      document.cookie = `threaderAuthToken=${authResult.user.xa}`;
+
+      return true;
     },
   },
+  signInSuccessUrl: '/',
```

Comment:
I don't think you are saving the correct thing? I believe you can just use:
```
  callbacks: {
    signInSuccessWithAuthResult: async () => {
      const threaderAuthToken = await firebase.auth().currentUser.getIdToken(false);
      document.cookie = `threaderAuthToken=${threaderAuthToken}`;
      return true;
    },
  },
```

Reply:
I think both those things are the exact same.

Reply:
yeah dw, This is referring to before the latest update